Koans/AboutEnumerations.cs

```diff
@@ -0,0 +1,134 @@
+                  Endor,
+                  Hoth
+            }
+
+            enum LogLevel
+            {
+                  Debug = 100,
+                  Warning = 200,
+                  Verbose = 300
+            }
+
+            [Flags]
+            enum DayOfTheWeek
+            {
+                  Monday,
+                  Tuesday,
+                  Wednesday,
+                  Thursday,
+                  Friday,
+                  Saturday,
+                  Sunday
+            }
+
+            [Step(1)]
+            public void CreatingAnInstance()
+            {
+                  /*
+                  Creating an instance of an enum is as easy as assigning a member
+                  of the enum to a new variable. For instance:
+                  */
+                  var dagobah = Planet.Dagobah;
+
+                  /*
+                  While the underlying type of the instance above will be an integer,
+                  enums are first class types in the C# language. The type of the
+                  variable 'dagobah' will be the enum itself.
+                  */
+                  Assert.Equal(typeof(FillMeIn), dagobah.GetType());
+
+                  /*
+                  Adding new members to an enum is straight-forward and as you'd expect.
+                  Try adding a new member to the 'Planet' enum called 'Tantooine'.
+                  */
+                  Assert.True(Enum.IsDefined(typeof(Planet), "Tantooine"));
+            }
+
+            [Step(2)]
+            public void CastingToAnEnum()
+            {
+                  /*
+                  Because the underlying type of enum members are actually integers,
+                  you can create instances of an enum by casting an integer value to
+                  an enum. 
+
+                  Note that the associated constant value of members start with zero
+                  and increase by one.
+                  */
+                  var forestPlanet = (Planet)1;
+                  Assert.Equal(FILL_ME_IN, forestPlanet);
+
+                  /*
+                  Why would casting integers to enums be valuable? You may want to
+                  save values to a database perhaps. Those values would be stored as
+                  an integer and when querying for those values, you'd be handed an
+                  integer back. For instance:
+
+                  var usersHomePlanet = (Planets)row['planet'];
+
+                  Handling integer values as enumerations improves readability.
+                  */
+            }
+
+            [Step(3)]
+            public void MemberValuesCanBeExplicit()
+            {
+                  /*
+                  There may be cases where you don't want enum members to start at zero
+                  and incrementing by one. Perhaps you want them incrementing by 100.
+                  */
+                  var logLevel = LogLevel.Verbose;
+                  Assert.Equal(300, (int)logLevel);
```

Comment:
Will this assert fail?

Reply:
I must have missed rolling back the solution during clean up. I'll fix this.

Reply:
You have no idea how many times I have done this exact thing. 😂